### ISSUE TITLE
Handle vetos

### ISSUE THREAD

**Issue by AnthonyLim23** [ISISMuon/MuonDataLib#19]

**Comment by AnthonyLim23** [comment]
when counting veto frames, should it include those that have been filtered?

If a frame is vetoed should it be counted as being filtered?

**Comment by wilkojohnny** [comment]
If you arrange your filters so that you keep all data between t1 and t2, then I would expect the raw_frames to be the number of frames between t1 and t2 completely ignoring any vetos. Then good_frames will be raw_frames minus the number of frames between t1 and t2 which have been vetoed.

**Comment by AnthonyLim23** [comment]
so the order of operation is filters then vetos?

The number of vetos is then the number of frames that have been removed (due to a veto flag) after the data has been filtered. Therefore, we can get the number of veto frames down to zero by using a filter to remove those frames first.

**Comment by wilkojohnny** [comment]
Yes, and I think that concurs with what we have now: With our current system, our 'filtering' is starting and stopping the DAE. The vetos then get dealt with while the run is ongoing.

I will just mention that if a frame is vetoed, theres a 99% chance the user will not want it -- so I would suggest that removing vetos happens automatically unless it is explicitly switched off.
